tests/test.js

```diff
@@ -0,0 +1,18 @@
+/* It sure is a sunny day! */
+
+function hello_word(name) {
+  console.log(`Hello ${name}`);
+}
+
+var SOME_RE = new RegExp("T[hat]{3}Xliner");
+
+var argv = process.argv.slice(2);
```

Comment:
I mean, Ok..... But is this necessary?

Reply:
According to [node's documentation](https://nodejs.org/en/docs/), Node has two arguments along with the arguments you type into the terminal. I remove those 2 arguments because they are unnecessary